mv/MBS_MobileDirPad.js

```diff
@@ -1,131 +1,477 @@
 //=============================================================================
-// MBS - Mobile Dir Pad (v1.2.0)
+// MBS - Mobile Dir Pad Upgraded
 //-----------------------------------------------------------------------------
 // por Masked
```

Comment:
You can add credits for you here if you want to, just put it like "reviewed/patched by Eliaquim"
I really didn't think it through though, because this would be hard to maintain if more people were to contribute xd

Anyway, feel free